pkg/kubevirt/types.go

```diff
@@ -16,8 +16,14 @@ package kubevirt
 
 import (
 	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ClientFunc creates a client based on the kubeconfig saved in the secret data.
+type ClientFunc func(secret *corev1.Secret) (client.Client, error)
+
```

Comment:
Please move that to `data_volume.go`, so far it's only used there. This `types.go` file is reserved for global constants, variables, and types that are used throughout the codebase.